test-requirements.txt

```diff
@@ -1,4 +1,5 @@
-molecule>=2.15.0,<3.0.0
+molecule>=3.0.0
```

Comment:
Will this work?

Reply:
We'll probably have to do some other work, but I want the skeleton updater to not try and downgrade stuff we've updated.

Reply:
ack, makes sense :+1: